grits.sh

```diff
@@ -63,7 +63,11 @@ export LOCAL_IP=$(ifconfig $ethernet|grep "inet addr"|awk -F":" '{print $2}'|awk
 inside_container sed -i "s/mongodb:\/\/CHANGEME/mongodb:\/\/$LOCAL_IP/" /source-vars.sh
 inside_container sed -i "s/redis:\/\/CHANGEME/redis:\/\/$LOCAL_IP/" /source-vars.sh
 inside_container sed -i "s/http:\/\/CHANGEME/http:\/\/$LOCAL_IP/" /source-vars.sh
+
+ECHO_STR="export BROKER_URL=redis://$LOCAL_IP:6379/0"
```

Comment:
Let's move it to compose. We should move towards doing configuration by passing in environment variables and mounting configuration files rather than running scripts.